services/user-api/lambdas/get.js

```diff
@@ -10,7 +11,7 @@ export const main = async event => {
   const { personalNumber } = event.pathParameters;
 
   const params = {
-    TableName: 'users',
+    TableName: config.users.TableName,
```

Comment:
Are AWS aware of this config params? How does these config vars get compiled with serverless?

Reply:
Asking this since i could not retrive a user from my database after sls deploy.

Reply:
Fixed typo. Should be config.users.tableName.